src/main/java/eu/ai4work/sws/service/SlidingDecisionService.java

```diff
@@ -0,0 +1,67 @@
+package eu.ai4work.sws.service;
+
+import eu.ai4work.sws.model.specific.CongestionLevel;
+import eu.ai4work.sws.model.specific.TruckPosition;
+import eu.ai4work.sws.model.generic.SlidingDecisionResult;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+@RequiredArgsConstructor
+public class SlidingDecisionService {
+    private final RuleEngineService ruleEngineService;
+    public Map<String, Object> getSlidingDecision(Map<String, Object> parameters) {
+        Map<String, Object> decisionResults = new HashMap<>();
+
+        int noOfTrucksInQueue = (int) parameters.getOrDefault("noOfTrucksInQueue", 0);
```

Comment:
The hardcoded string `"noOfTrucksInQueue"` should better be defined as `static final String`, same applies for the string in the next row.

Reply:
Done in the commit https://github.com/AI4WORK-Project/sliding-work-sharing/pull/2/commits/6e63b8bce37097860fbe069304276275bb28ba83